tests/subsys/scsi/scsi_ufs/testcase.yaml

```diff
@@ -0,0 +1,14 @@
+common:
+  depends_on:
+    - scsi
+  tags:
+    - scsi
+    - ufs
+  filter: dt_alias_exists("ufs0")
+tests:
+  scsi.ufs.scsi_basic:
+    depends_on: ufshc
+  scsi.ufs.scsi_rw:
```

Comment:
This kind of seems like it is going to end up with platforms excluding this case if they can't handle the RW test. Should we instead define one test and allow boards to opt in to the RW test using an overlay (or opt-out, if we want the test on by default)